clients/admin-ui/package.json

```diff
@@ -18,7 +18,8 @@
     "format": "prettier --write src __tests__/",
     "format:ci": "prettier --check src __tests__/",
     "lint": "eslint . --ext .ts,.tsx",
-    "lint:fix": "lint-staged --diff=main",
+    "lint:fix": "eslint . --fix --ext .ts,.tsx",
+    "lint-staged:fix": "lint-staged --diff=main",
```

Comment:
I ran `lint:fix` a few times expecting it to fix some of the automatic ones, but it didn't do anything!

Turns out `lint:fix` only operated on files that you had explicitly staged for commit with git. That's technically "more efficient" since it doesn't scan everything, but it really confused me for a solid while.

I suspect everyone else would make the same assumption I did, so I just changed it 😄

Reply:
ahh, that makes sense. Good update. I honestly had no idea it was only running on staged changes